app/controllers/bananas2_controller.rb

```diff
@@ -0,0 +1,78 @@
+class Bananas2Controller < ApplicationController
+  
+  
+  
+  before_action :set_banana, only: [:show, :edit, :update, :destroy]
+
+  # GET /bananas
+  # 
+  # GET /bananas.json
+  def index
+    @bananas = Banana.all
+  end
+
+  # GET /bananas/1
+  # GET /bananas/1.json
+  def show
+  end
+
+  # GET /bananas/new
+  def new
+    @banana = Banana.new
+  end
+
+  # GET /bananas/1/edit
+  def edit
+  end
+
+  # POST /bananas
+  # POST /bananas.json
+  def create
+    @banana = Banana.new(params[:banana])
+
+    respond_to do |format|
+      if @banana.save
+        format.html { redirect_to @banana, notice: 'Banana was successfully created.' }
+        format.json { render action: 'show', status: :created, location: @banana }
+      else
+        format.html { render action: 'new' }
+        format.json { render json: @banana.errors, status: :unprocessable_entity }
```

Comment:
__[staging][rubocop]__
* Metrics/LineLength: Line is too long. [82/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)